terraform/install.sh

```diff
@@ -75,22 +75,35 @@ getBillingAccount() {
 }
 
 getProject() {
-  log "Checking for project list"
+  log "Checking for project list..."
+  acct=$(gcloud info --format="value(config.account)")
+  # get projects with user account
+  owner_projects=$(gcloud projects list --filter="id:stackdriver-sandbox-* AND name='Stackdriver Sandbox Demo'" --format="value(projectId)")
+  # get projects with billing account
   billed_projects=$(gcloud beta billing projects list --billing-account="$billing_id" --filter="project_id:stackdriver-sandbox-*" --format="value(projectId)")
-  # only keep projects with name "Stackdriver Sandbox Demo"
+  declare -A proj_map
+  for proj in ${billed_projects[@]}
+  do
+    proj_map["$proj"]="yes"
+  done
```

Comment:
I don't think I understand the purpose of proj_map

Reply:
It's easier to check if a variable exists in a map than a list.

Reply:
It would be more readable if we only had one list of projects, and we could just iterate through the list. Something like this: https://stackoverflow.com/questions/2696055/intersection-of-two-lists-in-bash

But I'm skeptical that we really need both `billed_projects` and `owned_projects`